src/formatDate.js

```diff
@@ -51,6 +51,60 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const inputArray = date.split(fromFormat[fromFormat.length - 1]);
+
+  let d, m, y, separator, inputYearLength;
```

Comment:
Instead of this variables (except length) you can use one object with format elements as keys

Reply:
Also create more meaningful variables names instead of `d`, `m` or `y`